waffle_hub/hub/hub.py

```diff
@@ -676,12 +713,8 @@ def check_train_sanity(self) -> bool:
         Returns:
             bool: True if all files are exist else False
         """
-        if not (
-            self.model_config_file.exists()
-            and self.best_ckpt_file.exists()
-            # and self.last_ckpt_file.exists()
-        ):
-            raise FileNotFoundError("Train first! hub.train(...).")
+        if self.get_train_status()["status"] != "SUCCESS":
```

Comment:
Using Enum instance would be better choice.

Reply:
Right! I will add 'StatusCase' enum class.